cfme/tests/infrastructure/test_vmrc_console.py

```diff
@@ -1,85 +1,73 @@
 import pytest
 
 from cfme import test_requirements
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import providers
+from cfme.utils.providers import ProviderFilter
+
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
```

Comment:
Since you're module scoping your provider, you'll very likely want to use setup_provider_modscope.